MariaKarpava/units-tracking-app
# Add home screen implementation

Summary of changes

- Get rid of drinkService environment object in order to stop all screens being updated when drinks array changes (observed object was in the very root of the hierarchy).
- Implement MVVM for Home screen.
- Link DrinkService with HomeViewModel via NotificationCenter so that when drinks array changes it posts notification to HomeViewModel and triggers its update.-
- Add fonts from Figma.
- Add color & font extensions which provide semantic naming.

Test Plan

Note: units = strength (ABV, %) x volume (ml) ÷ 1,000

Test 1 - When adding new drink then remaining count on Home is updated. 
1) go to AddNewDrink screen and choose ABV and ml, tap add button
2) go to HomeScreen and check how remaining number of units has changed


Test 2 - Check colors of remaining units.
1) go to AddNewDrink screen and add new drink.
2) go to HomeScreen and check the color of units:
2) when remaining number of units is more than 3 then the color of units should be grey/black.
3) when remaining number of units is less than 3 then the color of units should be orange.
4) when remaining number of units is zero then display the warning Image.
